app/assets/javascripts/application.js

```diff
@@ -104,4 +104,54 @@ function validateNewsLetterEmail(event) {
     $('#nl_message').html("");
     $('#nl_message').hide();
    }
+}
+
+function veriryNewsLetterEmailInSignUpPage(val, email, alt_email) {
+  // If alternate email is present in sign-up page take alternate-email to send news-letters other-wise DOE email.
+  news_letter_email = alt_email
+  if (news_letter_email == ""){
+    news_letter_email = email
+  }
+  $('#news-letter-ajax-message').show();
+  $('#news-letter-ajax-message').html('Verifying email')
+  $('#news-letter-ajax-message').css("color", '#c4262d');
```

Comment:
This looks like a dark red for the message letters, which is in our color scheme, but might not be super-visible.  Maybe a good place to take note for the future redesign, I'd love to see our error messaging be easier to absorb.

Reply:
Yes will take a note